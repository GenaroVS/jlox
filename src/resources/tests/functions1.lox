fun sayHi(first, last) {
    print "Hi, " + first + " " + last + "!";
}
sayHi("Dear", "Reader");
sayHi("Guillermo" + " de", "Torro");
sayHi("Dear", nil);
