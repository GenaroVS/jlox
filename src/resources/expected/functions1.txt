Hi, Dear Reader!
Hi, Guillermo de Torro!
[line 2] ERROR: Operands must be two numbers or two strings.
